CONTRIBUTING.md

```diff
@@ -0,0 +1,276 @@
+
+```text
+<type>([scope]): <subject>
+[BLANK LINE]
+[body]
+[BLANK LINE]
+[footer]
+```
+
+> **Note**
+> The type and subject are mandatory.
+
+Examples include:
+
+```text
+feat: add support for forum channels
+```
+
+```text
+refactor(commands)!: change name of Bot to CommandBot
+
+This is a breaking change because the name of the class has changed.
+```
+
+```text
+fix(app-cmds): resolve issue with negative numbers not converting to integers
+
+Co-Authored-By: Some Person <email@example.com>
+```
+
+### Type
+
+`<type>` is one of the following:
+
+- **build**: Changes that affect the build system or external dependencies
+- **ci**: Changes to our CI configuration files and scripts such as GitHub Actions
+- **docs**: Documentation only changes
+- **feat**: A new feature
+- **fix**: A bug fix
+- **perf**: A code change that improves performance
+- **refactor**: A code change that neither fixes a bug nor adds a feature
+- **style**: Changes that do not affect the meaning of the code (white-space, formatting, etc)
+- **test**: Adding missing tests or correcting existing tests
+
+### Scope
+
+`[scope]` is the name of the module affected (as perceived by the person reading the changelog generated from commit messages). Scope is not required when the change affects multiple modules. Some examples are:
+
+- `application_command.py` - `app-cmds` is the scope
+- `auto_moderation.py` - `automod` is the scope
+- `ext/commands/*` - `commands` is the scope
+- `ui/` - `ui` is the scope
+
+### Subject
+
+The subject should be a short summary of the commit, the body can be used for more info, so do not cram so much into the subject. Ideally this should be 50 characters or less, but it is not a hard limit, 72 characters is fine if necessary, such as if a long method name is described.
+
+### Body
+
+The body is an optional long description about the commit, this can be used to explain the motivation for the change, and can be used to give more context about the change. It is not required, but it is recommended.
+
+### Footer
+
+The footer contains optional metadata about the commit. These are sometimes added by git or similar tools, and examples include `Co-authored-by`, `Signed-off-by`, `Fixes`.
+
+## Documentation
+
+Nextcord uses [Sphinx](https://www.sphinx-doc.org/en/master/) for documentation compilation. Specifically, we use the numpy format ([Reference](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard), [Examples](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html)).
+
+### Building the Docs
+
+To install the documentation dependencies, run `pip install -r docs/requirements.txt` in the root of the project.
+
+To build the docs, run `make html` (`./make html` on Windows, requires [Make](https://www.gnu.org/software/make/) on linux). If you don't have make, you can use `python -m sphinx -b html docs docs/_build/html` instead) in the `docs` directory.
```

Comment:
Shouldn't Linux be capitalized here? It's a proper noun.

Reply:
```suggestion
To build the docs, run `make html` (`./make html` on Windows, requires [Make](https://www.gnu.org/software/make/) on Linux). If you don't have make, you can use `python -m sphinx -b html docs docs/_build/html` instead) in the `docs` directory.
```